docs/emu_mps/notebooks/getting_started.ipynb

```diff
@@ -196,18 +196,15 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 6,
+   "execution_count": null,
```

Comment:
do we really need to modify this file?

Reply:
this `null` happened automatically because I didn't run the file.

I've modified other part. The basis "r,g" was a `set` `{"r", "g"}`. The pulser constructor accepts `Sequence` but not a set

Reply:
ok